src/app/shared/state/checker-workflow.query.ts

```diff
@@ -44,11 +45,29 @@ export class CheckerWorkflowQuery extends Query<CheckerWorkflowState> {
    * Returns true if entry is a workflow, returns false if entry is not a workflow (i.e. a tool)
    * @param entry The entry to check
    */
-  public isEntryAWorkflow(entry: Entry): boolean {
+  public isEntryAWorkflow(entry: Entry | null): boolean {
```

Comment:
If you're declaring the parameter can be null, then shouldn't you add to the return type as well? Not sure if this is common or not.

Also, although it was like this already, I find it slightly weird that a function named like a boolean function and returning a boolean can return a `null`. It's really a function that returns 3 states.